specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/stable/2021-10-01/examples/UpdateClusterExample.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "api-version": "2021-10-01",
+    "subscriptionId": "1bfbb5d0-917e-4346-9026-1d3b344417f5",
+    "resourceGroupName": "k8sc-rg",
+    "clusterName": "testCluster",
+    "ConnectedClusterPatch": {
+      "tags": {
+        "tag1": "value1",
+        "tag2": "value2"
+      },
+      "properties": {
```

Comment:
Let's remove this property bag from the example. We wil actually be throwing error in this case

Reply:
Removed it